apel-ssm.spec

```diff
@@ -21,7 +21,7 @@ BuildArch:      noarch
 BuildRequires:  python-devel
 %endif
 
-Requires:       stomppy < 4.0.0, python-daemon, python-dirq, python-ldap
```

Comment:
Should this now be `stomppy >= 3.1.6` if earlier ones are no longer supported?

Reply:
I've added the restriction `stomppy >= 3.1.1`, as that version is pip-installable and has been tested to be able to send and receive messages.